examples/xgboost/train.py

```diff
@@ -43,6 +45,7 @@ def main():
         params = {
             'objective': 'multi:softprob',
             'num_class': 3,
+            'learning_rate': args.eta,
```

Comment:
curious why we made the parser argument "eta" and use "learning_rate" to specify the parameter to xgboost. they are interchangeably used by xgboost, right? is "eta" more commonly used by xgboost users?

Reply:
@sueann, I have changed the argument from "eta" to "learning_rate" and No, "learning_rate" is commonly used by users. I think now it's looking good after the change.